### ISSUE TITLE
Option to redirect frontend to external host

### ISSUE THREAD

**Issue by rogierlommers** [containous/traefik#119]
With haproxy, I'm able to create redirects. For example, see the following snipplet:

`redirect prefix http://www.redirectotthis.com code 301 if {hdr(host) -i requestedsite.com}`

If someone is requesting `requestedsite.com`, he gets redirected to an externally hosted site: `redirecttothis.com`. It would be great if traefik also supports this. So instead of providing a backend in the frontend, there could be an option `redirect = "http://redirecttothis.com"`. What do you think?

**Comment by emilevauge** [comment]
Hey @rogierlommers, thank for your proposal :) How would you configure it in the `traefik.toml` configuration file (as an example)?

**Comment by rogierlommers** [comment]
Well, not exactly sure, because it would be great to be backwards compatible. Something like this?

```[frontends]
  [frontends.redirect1]
  backend = redirect
    [frontends.frontend1.routes.test_1]
    rule = "redirect"
    value = "http://page-to-be-redirected-to.com"```

**Comment by gtrummell** [comment]
/bump A non-proxying redirect would be fantastic!  At this very moment I'm being forced to use HAProxy with consul-template to do this, but I am about to use Traefik everywhere else.

**Comment by gaetancollaud** [comment]
I find this feature very useful ! 

In my case I have some old sub-domains that I want to redirect to other ones.

**Comment by errm** [comment]
Sounds like a nice feature to have ... does anyone plan to work on this?

**Comment by rogierlommers** [comment]
It's on my todo list; but I'm kind of running out of time...
Op di 13 sep. 2016 om 16:11 schreef Ed Robinson <notifications@github.com>

**Comment by robbert229** [comment]
Is this still on your todo list? This is a nice feature that I could definitely use.

**Comment by ldez** [comment]
Closed by #178 (redirect by entrypoint)
